src/test/java/friday/DeadlineTest.java

```diff
@@ -0,0 +1,30 @@
+package friday;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import java.time.format.DateTimeParseException;
+
+public class DeadlineTest {
+
+    /**
+     * Tests that valid date formats are correctly parsed and represented in the Deadline class.
+     */
+    @Test
+    public void testValidDateFormats() {
+        Deadline deadline1 = new Deadline("Test Task", "1/12/2019");
+        assertEquals("[D][ ] Test Task (by: Jan 12 2019)", deadline1.toString());
+    }
+
+    /**
+     * Tests that an invalid date format throws the expected DateTimeParseException with a relevant error message.
+     */
+    @Test
+    public void testInvalidDateFormat() {
```

Comment:
Should this test method be named like `testDeadline_invalidDate_exceptionThrown`? I am seeing this issue on naming for the different test methods in other parts of the code.